admin/environment-management/ssh-access.md

```diff
@@ -1,17 +1,114 @@
 ---
-title: "Disable SSH Access"
-description: Learn how to disable SSH to comply with ingress restrictions.
+title: "SSH Configuration"
+description: Learn how to configure SSH access to Coder environments.
 ---
 
-By default, Coder enables SSH access for all users. There are no additional
-dependencies required.
+By default, Coder enables SSH access for all users. Coder assigns each user a
+private key that they can use to access their environments.
+
+## Background
+
+Part of the standard Coder environment asset bundle is a lightweight SSH server
+mounted onto the environment agent; the lightweight SSH server is a backup used
+when Coder can't find a server available on port 22. This allows slimmer images
+to remain accessible via SSH without the need for additional image dependencies.
+
+## Using OpenSSH
+
+The built-in SSH server is limited, and does not implement
+advanced functionality like X11 forwarding or `sshd_config` specifications.
+If SSH is the primary mode of access to Coder for your users, consider
+running a full OpenSSH server with `systemd` inside your image instead.
+
+To do so:
+
+1. Make sure that you're creating your environments with the [CVM
+   option](https://coder.com/docs/environments/cvms) enabled
+2. Add the following to your Dockerfile:
```

Comment:
Change the order of the listed items. Need image built before environment creation.